exercises/01/readme.md

```diff
@@ -81,6 +81,21 @@ When successfully installed, you should see the extension thus:
 
 ![CDS Language Support extension installed in VS Code](vscode-extension.png)
 
+To be able to open VS Code from the usual OS command line you need to add the installation path of VS Code to the environment variable PATH. There's an option to do this via the `Command Palette` in VS code. 
+
+Open the `Command Palette` by pressing `Ctrl+Shift+P` in VS Code or via the menu bar.
```

Comment:
- how about on macOS? 
- is "P" really upper case in "Ctrl-Shift-P"? I think the convention is to show the keys, i.e. Ctrl-Shift-p
- via the menu bar -> via menu path View -> Command Palette

Reply:
done!